sdk/keyvault/keyvault-keys/src/index.ts

```diff
@@ -903,4 +982,17 @@ export class KeysClient {
 
     return resultObject;
   }
+
+  /**
+   * Creates a span using the tracer that was set by the user
+   * @param methodName The name of the method for which the span is being created.
+   * @param requestOptions The options for the underlying http request. This will be
+   * updated to use the newly created span as the "parent" so that any new spans created
+   * after this point gets the right parent.
+   */
+  private createSpan(methodName: string, requestOptions: RequestOptionsBase): Span {
+    const span = TracerProxy.getTracer().startSpan(methodName, requestOptions.spanOptions);
+    requestOptions.spanOptions = { ...requestOptions.spanOptions, parent: span };
```

Comment:
It'd be nice if we could do this without directly manipulating the original `requestOptions` object, but it's fine for now.